.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,58 @@
+# This is a GitHub workflow defining a set of jobs with a set of steps.
+# ref: https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions
+#
+# This test is meant to verify that our Dockerfile can build successfully when
+# changed on the CPU architectures we have made it support being built on.
+#
+name: Build ibm-powervs-block-csi-driver-centos-base
+
+on:
+  push:
+    paths:
+      - "build/base/Dockerfile"
+      - ".github/workflows/docker-build.yml"
+
+jobs:
+  docker:
+    name: Build ibm-powervs-block-csi-driver-centos-base
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      # Action reference: https://github.com/docker/setup-qemu-action
+      - name: Set up QEMU (for docker buildx)
+        uses: docker/setup-qemu-action@v1
+
+      # Action reference: https://github.com/docker/setup-buildx-action
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      # Action reference: https://github.com/docker/login-action
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # Convert repository owner name into lowercase letters
+      # Action reference: https://github.com/ASzc/change-string-case-action
+      - id: repository_owner
+        name: lower case repository owner name
+        uses: ASzc/change-string-case-action@v1
+        with:
+          string: ${{ github.repository_owner }}
+
+      # Action reference: https://github.com/docker/build-push-action
+      - name: Build container
+        uses: docker/build-push-action@v2
+        with:
+          context: ./build/base
+          platforms: linux/amd64,linux/ppc64le
+          target: centos-base
+          push: true
+          tags: ghcr.io/${{ steps.repository_owner.outputs.lowercase }}/ibm-powervs-block-csi-driver-centos-base:V8.0
```

Comment:
@mkumatag is it okay to hardcode the base image version?

Reply:
The value is hardcoded here: https://github.com/kubernetes-sigs/nfs-ganesha-server-and-external-provisioner/blob/abfbe652e38e5a0562c70ce1af517a31a07d7c9a/.github/workflows/docker-build.yml#L50

Reply:
I'm okay having it here as well, other option would be like reading from some file.

Reply:
can you rename rename the tag to `v8.0.0`(lowercase V)

Reply:
fixed